reconcile/gitlab_housekeeping.py

```diff
@@ -0,0 +1,73 @@
+import logging
+
+from datetime import datetime, timedelta
+
+from utils.config import get_config
+from utils.gitlab_api import GitLabApi
+
+
+def get_housekeeping_gitlab_api():
+    config = get_config()
+
+    gitlab_config = config['gitlab']
+    server = gitlab_config['server']
+    token = gitlab_config['token']
+    project_id = gitlab_config['housekeeping']['project_id']
+
+    return GitLabApi(server, token, project_id=project_id, ssl_verify=False)
```

Comment:
We can perhaps have `ssl_verify` set to true now that we have internal certs trusted :) 

But okay to leave it at

Reply:
lets keep this PR self contained. wdyt?